api/SWR/CarePackage/useCarePackageApi.js

```diff
@@ -28,7 +28,43 @@ const useCarePackageApi = {
     useGetData(shouldFetch ? `/suppliers${getQueryParamsFromObject({ supplierName })}` : null),
 
   singleSupplier: (supplierId) => useGetData(supplierId ? `/suppliers/${supplierId}` : null, '', {}),
-  singlePackageInfo: (packageId) => useGetData(`${CARE_PACKAGES_URL}/${packageId}`, '', {}),
+  singlePackageInfo: (packageId) => {
+    const response = useGetData(`${CARE_PACKAGES_URL}/${packageId}`, '', {});
+    return { ...response, singlePackageInfoLoading: !!response.data};
+  },
+  serviceUser: ({ hackneyId }, shouldFetch) => (
+    useGetData(shouldFetch ? `/service-user/${hackneyId}` : null, 'Can not get service user', {})
+  ),
+  serviceUserSearch: ({
+    firstName,
+    lastName,
+    hackneyId,
+    dateOfBirth,
+    postcode,
+    pageNumber,
+    pageSize,
+    orderBy
+  }, shouldFetch) => (
+    useGetData(shouldFetch ? `/service-user/search${getQueryParamsFromObject({
```

Comment:
About query params for that and other requests
`useGetData` was supposed to be used for simple requests without any params. And although the current approach is working with this query builder - this is not how it is supposed to be done from the SWR perspective.

https://swr.vercel.app/docs/arguments#multiple-arguments
Here is an example of how it should be done for multiple arguments. I think we should consider refactoring it in future
cc @akeamwoodsND